### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by hjoneweek**
Updated addpatient and addrecord parts of the user guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -75,20 +75,39 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a patient: `addpatient`
 
-Adds a person to the address book.
+Enables the user to add a patient's contact and medical details.
```
Comment (Darren159) [inline docs/UserGuide.md:80]: Dont need "Enables the user to", just "Adds a patient's contact and medical details.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -75,20 +75,39 @@ Shows a message explaning how to access the help page.
...
+Expected Outputs on Success
+* **GUI:** Contact details added to the respective patient's record.
+* **Message:** Patient details added successfully.
+
+Expected Output on Failure
+* **Message:** Failed to add details. Please check the command format and try again.
+
+### Adding Medical Records : 'addrecord'
+
+Allows the user to add a new patient's medical records to the app.
```
Comment (Darren159) [inline docs/UserGuide.md:96]: Same as above

**Review by Darren159 — COMMENTED** [review]
LGTM, just some minor changes.

**Comment by hjoneweek** [comment]
Changes made.

**Review by Darren159 — APPROVED** [review]
LGTM

**Pull request merged**